media_driver/cmake/linux/media_compile_flags_linux.cmake

```diff
@@ -176,6 +176,10 @@ set(MEDIA_COMPILER_FLAGS_DEBUG
     -DINSTR_GTUNE_EXT
 )
 
+if(X11_FOUND)
+    add_definitions(-DX11_FOUND)
```

Comment:
To me passing -DHAVE_X11=1 seems better since it is aligned with what people are accustomed to with autotools. In this case Android should explicitly set -DHAVE_X11=0 and checks below should look like:
`#if HAVE_X11`